packages/cashier/src/stores/account-transfer-store.ts

```diff
@@ -467,9 +466,11 @@ export default class AccountTransferStore {
 
                 const { account_id, login } = this.root_store.traders_hub?.selected_account;
 
+                const is_traders_hub = window.location.pathname === routes.traders_hub;
+
                 //if from appstore -> set selected account as the default transfer to account
                 //if not from appstore -> set the first available account as the default transfer to account
-                if (!is_from_pre_appstore || [account_id, login].includes(account.loginid)) {
+                if (is_traders_hub || [account_id, login].includes(account.loginid)) {
                     this.setSelectedTo(obj_values);
```

Comment:
@thisyahlen-deriv should be `!is_traders_hub` instead of `is_traders_hub`.

Reply:
Will use @mahdiyeh-deriv 's suggestion 🙏